doc/sphinx-guides/source/admin/big-data-administration.rst

```diff
@@ -0,0 +1,312 @@
+
+   * Use a Globus Store
+   * Work with users to size files appropriate to the underlying storage
+   * Consider Globus over S3 when normal upload/download options (via the UI/API) are desired along with Globus transfer
+
+5. **For Petascale datasets, or extreme numbers of files:**
+
+   * Consider a Remote Store and referencing a single URL/Globus endpoint for the entire dataset
+
+
+Managing more files per dataset and more datasets
+-------------------------------------------------
+
+Dataverse can be configured to handle datasets with hundreds or thousands of files and hundreds of thousands of datasets. However, reaching these levels can require significant effort to appropriately configure the server.
+
+Technically, there are two factors that can limit scaling the number of files per dataset: how much the Dataverse server is involved in data transfer, and constraints based on Dataverse's code and database configuration.
+The former is dramatically affected by the choice for file storage and options such as the S3 direct upload/download settings and ingest size limits. There are fewer ways to affect the latter beyond increasing the amount of memory and CPU resources available
+or rewriting the relevant parts of Dataverse. (There are continuing efforts to improve Dataverse's performance and scaling, so it is also advisable to use the latest version if you are pushing the boundaries on scaling. Progress is being made.)
+
+Scaling to larger numbers of datasets (and to some extent scaling files per dataset) also depends on Dataverse's Solr search engine. There have been very significant improvements in indexing and search performance in recent releases, including some that are not turned on by default.
+
+Avoiding Many Files
+~~~~~~~~~~~~~~~~~~~
+
+Before describing things that can be done to improve scaling, it is important to note that there are configuration options and best practices to suggest to users to help avoid larger datasets and and help them avoid hitting performance issues by going beyond the file counts you know work in your instance.
+
+There are a number of settings to limit how many files can be uploaded (see :ref:`database-settings` and :ref:`jvm-options` for more details):
+
+- ZipUploadFilesLimit - the maximum number of files allowed in an uploaded zip file - only relevant for file stores and S3 when direct upload is not used.
+- MultipleUploadFilesLimit - the number of files the GUI user is allowed to upload in one batch, via drag-and-drop, or through the file select dialog
+- MaxFileUploadSizeInBytes - limit the size of files that can be uploaded
+- UseStorageQuotas - once enabled, super users can set per-collection quotas (in bytes) to limit the aggregate size of all files in the collection
+- dataverse.files.default-dataset-file-count-limit - directly limits the number of files per dataset, can be changed per dataset via API (by super users)
+
+Ways to Scale
```

Comment:
The anchor is big-data-administration.html#ways-to-scale

It might be better to have an anchor that's more particular to this section.

Reply:
Changed to Scaling-related Configuration